### PR TITLE
PCX-1806: Support ExtraElements in payment list

### PR DESCRIPTION
**Pull request by siebeprojects**

### CODE REVIEW

File: checkout/src/main/java/com/payoneer/checkout/model/Checkbox.java [hunk 1]
```diff
@@ -10,18 +10,18 @@
...
-    /** Advanced API, required */
-    private String name;
-    /** Advanced API, required */
+    /** Defines the mode of this Checkbox, required */
     @CheckboxMode.Definition
     private String mode;
-    /** Advanced API, optional */
-    private String requireMsg;
+    /** Error message that should be displayed if required checkbox is not checked by customer. */
+    private String requireMessage;
```
Comment (bzlat) [inline checkout/src/main/java/com/payoneer/checkout/model/Checkbox.java:26]: If you'd like to be absolutely `1-to-1` with the backend models, this should be named `requiredMessage`.

File: checkout/src/main/java/com/payoneer/checkout/ui/widget/NetworkLogosWidget.java [hunk 2]
```diff
@@ -20,43 +20,62 @@
...
+        logos.clear();
+        layout.removeAllViews();
 
-        LinearLayout layout = parent.findViewById(R.id.layout_logos);
         for (PaymentNetwork network : networks) {
             addNetworkLogo(network, layout);
         }
     }
 
+    public void setSelected(String networkCode) {
```
Comment (bzlat) [inline checkout/src/main/java/com/payoneer/checkout/ui/widget/NetworkLogosWidget.java:71]: Is this a `setSelected` or more of a `showSelected`?

**Review by bzlat — DISMISSED** [review]

File: checkout/src/main/java/com/payoneer/checkout/model/Checkbox.java (hunk 1 above)
Comment (siebeprojects) [reply checkout/src/main/java/com/payoneer/checkout/model/Checkbox.java]: Fixed this as it should be requiredMessage

File: checkout/src/main/java/com/payoneer/checkout/ui/widget/NetworkLogosWidget.java (hunk 2 above)
Comment (siebeprojects) [reply checkout/src/main/java/com/payoneer/checkout/ui/widget/NetworkLogosWidget.java:71]: I leave it to setSelected.

**Review by bzlat — APPROVED** [review]

**Pull request merged**